components/exercises/chapter-5-oscar/Everything/some.tsx

```diff
@@ -0,0 +1,10 @@
+/**
+ * Checks if at least one element in the array satisfies the provided testing function.
+ * @param array - The array to iterate over.
+ * @param callback The testing function.
+ * @returns `true` if at least one element satisfies the testing function, otherwise `false`.
+ */
+export const some = (
+  arr: number[],
+  callback: (arg: number) => boolean
+): boolean => !arr.some((n) => !callback(n));
```

Comment:
I don't think you are allowed to use `Array.some` here, the point of this exercise is to implement your own version using a loop

Reply:
literally says "using the some method"